core/templates/components/skiplinks/skiplinks.twig

```diff
@@ -0,0 +1,7 @@
+<header>
+  <a href="#main-content" class="su-skiplinks">{{ text|default('Skip to content') }}</a>
```

Comment:
Id's are hard and don't really belong in a pattern library. There is no way to ensure that the next developer's software doesn't or does have a main-content id. Unfortunately, for this functionality, we need one. I would suggest opening up the anchor link to a twig variable and perhaps generate a random id if one is not provided.

Example not tested*
```
{% set skipID = "skip-to-content-" ~ random() %
 <a href="{{ anchor|default(skipID) }}" class="su-skiplinks">{{ text|default('Skip to content') }}</a>
```